addOns/authhelper/src/main/java/org/zaproxy/addon/authhelper/AuthUtils.java

```diff
@@ -190,20 +202,33 @@ public static boolean authenticateAsUser(
             if (!userAdded) {
                 LOGGER.debug("Entering user field on {}", wd.getCurrentUrl());
                 userField.sendKeys(username);
+                if (demoMode) {
+                    sleep(2000);
+                }
             }
             try {
                 LOGGER.debug("Submitting password field on {}", wd.getCurrentUrl());
                 pwdField.sendKeys(password);
+                if (demoMode) {
+                    sleep(2000);
+                }
                 pwdField.sendKeys(Keys.RETURN);
             } catch (Exception e) {
                 // Handle the case where the password field was present but hidden / disabled
                 LOGGER.debug("Handling hidden password field on {}", wd.getCurrentUrl());
                 userField.sendKeys(Keys.RETURN);
+                if (demoMode) {
+                    sleep(2000);
+                }
                 sleep(TIME_TO_SLEEP_IN_MSECS);
                 pwdField.sendKeys(password);
+                if (demoMode) {
+                    sleep(2000);
+                }
```

Comment:
Is this because the dev add-on can be slow?

Do these need to stay? Should it be named something else?